modules/win_disable_services/manifests/disable_windows_update.pp

```diff
@@ -3,89 +3,97 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 class win_disable_services::disable_windows_update {
+  $win_update_key    = "HKLM\\SOFTWARE\\Policies\\Microsoft\\Windows\\WindowsUpdate"
+  $win_update_au_key = "${win_update_key}\\AU"
+  $win_au_key        = "HKLM\\SOFTWARE\\Microsoft\\Windows\\Windows\\AU"
+  service { 'wuauserv':
+    ensure => stopped,
+    name   => 'wuauserv',
+    enable => false,
+  }
+  case $facts['kernelrelease'] {
```

Comment:
Let's take another look at this. I would like avoid using $facts['kernelrelease'] , and it will need to be consistent through out puppet.